index.html

```diff
@@ -102,7 +102,7 @@ <h2 class="title">{{title}}</h2>
     <script src="data/githubToken.js"></script>
     <script src="scripts/model/index.js"></script>
     <script src="scripts/model/repos.js"></script>
-    <script src="scripts/view/filters.js"></script>
+    <script src="scripts/view/projectView.js"></script>
     <script src="scripts/view/repoView.js"></script>
     <script src="scripts/controller/projectsController.js"></script>
     <script src="scripts/controller/aboutController.js"></script>
```

Comment:
I would rename your page_routes.js to pageRoutes.js if you want to go with camelcase naming conventions for files. Just keep it consistent!